src/app/find/locationDetails/locationDetails.css

```diff
@@ -18,24 +18,37 @@
 .categoryCard {
   background-color: #f8f8fc;
   border: 1px solid #dadada !important;
-  padding: 4vw 4vw 4vw 4vw;
+  border-radius: 2px;
   box-shadow: 0vw 2vw 2vw rgba(0,0,0,0.15) !important;
   margin-bottom: 4vh;
   margin-top: 2vh;
 }
 
+.shelterService:last-of-type {
+  border-bottom: none;
+}
+
+.shelterService i {
```

Comment:
I'd tend to avoid giving custom styles to the actual `i` elements within the `Icon` component. It can break consistency between different instances of the atomic components if their styles are manipulated directly rather than through the standardized props.

Would it be possible to just wrap the `Icon` in a `span` or `div` and style those instead?